README.md

```diff
@@ -1,3 +1,1030 @@
 # Your first Wagtail site
 
-The site you build with Wagtail’s official tutorial: [Your first Wagtail site](https://docs.wagtail.org/en/stable/getting_started/tutorial.html).
+Starts from Wagtail’s getting started tutorial ([1a72d82](https://github.com/thibaudcolas/your-wagtail-portfolio/commit/1a72d82)), ends with a live site! (TODO)
+
+## New tutorial sections
+
+### Customize the home page
+
+Commit: [5081fda](https://github.com/thibaudcolas/your-wagtail-portfolio/commit/5081fda) Add new homepage fields
+
+#### Add a field for a headshot or graphic, a field for text that can include a short bio or enticing “hire me” text.
+
+To highlight:
+
+- `hero_cta_link`, a `ForeignKey` to a Page instance
+    - Explain `on_delete` and `related_name` if not explained before
+- `MultiFieldPanel` to group related fields
+- `firstof` template tag usage to display `hero_cta` if set, otherwise linked page’s title
+
+In `home/models.py`, add:
+
+```python
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
+
+
+class HomePage(Page):
+    # Hero section of HomePage
+    image = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Homepage image",
+    )
+    hero_text = models.CharField(
+        blank=True,
+        max_length=255, help_text="Write an introduction for the site"
+    )
+    hero_cta = models.CharField(
+        blank=True,
+        verbose_name="Hero CTA",
+        max_length=255,
+        help_text="Text to display on Call to Action",
+    )
+    hero_cta_link = models.ForeignKey(
+        "wagtailcore.Page",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        verbose_name="Hero CTA link",
+        help_text="Choose a page to link to for the Call to Action",
+    )
```

Comment:
If we added `hero_cta_url = models.URLField(...)` we could demonstrate how to use properties in a Page model, such as: 

```python
@property 
def hero_link(self):
    if self.hero_cta_link:
        return self.hero_cta_link.url 
    return self.hero_cta_url 
```